scripts/jenkins/unit-test.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -exuo pipefail
+
+export GOPATH=$WORKSPACE
+export PATH=$PATH:$GOPATH/bin
+eval "$(gvm ${GO_VERSION})"
+echo "Installing hey-apm dependencies and running unit tests..."
+go get -v -u github.com/golang/dep/cmd/dep
+go get -v -u github.com/graphaelli/hey/requester
```

Comment:
same as before. `dep ensure -v` and remove all `go gets` except the first one

Reply:
ditto